src/metricscollector/README.md

```diff
@@ -38,6 +41,9 @@ server:
   port: 8080
 logging:
   level: "info"
+db:
+  policy_db_url: "postgres://pqgotest:postgres@localhost/pqgotest" 
```

Comment:
my mistake...  will make the sample the same as the instructions in initializing db